tron/config/config_utils.py

```diff
@@ -157,6 +157,18 @@ def valid_name_identifier(value, config_context):
     return '%s.%s' % (config_context.namespace, value)
 
 
+def valid_context_variable_expr(command):
+    """ This funtion checks wrong format like '$(A) $(A)s' """
```

Comment:
*function

Also, we can be more specific about what "wrong" means. Something like, "verifies that all context variables have complete string format specifiers"?

Reply:
Yeah, that's better, thx.